iati/core/default.py

```diff
@@ -34,6 +35,23 @@ def get_default_version_if_none(version):
     return version
 
 
+def get_versions_by_integer():
```

Comment:
Suggest changing to a function with the prototype:

`def versions_for_integer(major_version) -> list of str`

The current invocation where you have to do square brackets afterwards to extract the info seems clunky.

Reply:
Good idea, this also makes the API and tests a lot simplier.